cmd/juju/common/model.go

```diff
@@ -41,7 +42,7 @@ type ModelMachineInfo struct {
 
 // ModelStatus contains the current status of a model.
 type ModelStatus struct {
-	Current        status.Status `json:"current" yaml:"current"`
+	Current        status.Status `json:"current,omitempty" yaml:"current,omitempty"`
```

Comment:
This has to be omitempty to cater for empty Status struct coming from apiserver. This change is correct and stands. Otherwise, we'll end up printing empty status which is not pretty.